golang/cosmos/x/vbank/vbank.go

```diff
@@ -241,5 +243,7 @@ func (ch portHandler) Receive(ctx *vm.ControllerContext, str string) (ret string
 }
 
 func (am AppModule) PushAction(ctx sdk.Context, action vm.Jsonable) error {
+	ctx = ctx.WithContext(context.WithValue(ctx.Context(), baseapp.TxHashContextKey, "x/vbank"))
```

Comment:
Document the *heck* out of in-band signaling like this.

Reply:
What do you mean by in-bound signaling? This is a placeholder value for the transaction hash since no real transaction exists.

Reply:
Yes, the use of a special string in a place where transaction hashes go is exactly "in-band signaling", which is a notorious source of errors, so should at least get a prominent comment where the field or key is defined so consumers know what to expect.

Reply:
I believe I added sufficient comments, let me know if I should further improve.

Reply:
I understand the concern about abusing a context key defined in a lower layer, but given this is only used to push a value into the directly called function, I would say it's a trade-off I'd prefer living with for now.

@michaelfig also mentioned that this whole pushed vbank balance update logic will be going away, so I'm not too worried about having this temporary cruft.